README.md

```diff
@@ -49,11 +49,7 @@ Build Instructions for Master
 5. Open `Blockzilla.xcodeproj` in Xcode.
 6. Build the `Focus` scheme in Xcode.
 
-V8.0 Development Branch
-----------------
-
-For version 8.0 of Focus, we are working off of the v8.0-dev branch.
-
-This branch is written in Swift 4.2. Pull requests for this branch must also be written in Swift 4.2. 
+Upcoming Release (Version 8.0)
+------------------
 
-For bugs and features for the upcoming v8.0 release, please see the V8.0 milestone within the GitHub Issues.
+Features planned for the next release can be found in the [V8.0 milestone](https://github.com/mozilla-mobile/focus-ios/milestone/29). If you would like to work on a feature that is not in this milestone, please reach out to @sblatz to ensure the proper UX and design work has been completed for it. That being said, all bug fixes or issues labeled [help wanted](https://github.com/mozilla-mobile/focus-ios/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) are fair game. 
```

Comment:
I'm not 100% sure the part about reaching out to you for anything not in the milestone is entirely necessary, but it's also not bad. Historically we've taken any contributor PRs that are based off previously filed issues, and if the UX work hasn't been done, we usually get tweaks in the PR review process. I suspect it would be more work to try to preemptively get UX work when a contributor is interested in an issue than just to edit/create when the PR has actually been submitted (obviously at least for things in reserve backlog).